cmd/ctr/pprof.go

```diff
@@ -1,6 +1,7 @@
 package main
 
 import (
+	"errors"
```

Comment:
@kevinetc123 he just means instead of using the stdlib `errors` use `github.com/pkg/errors`